packages/go_router/lib/src/router.dart

```diff
@@ -54,6 +57,7 @@ class GoRouter extends ChangeNotifier with NavigatorObserver {
     List<NavigatorObserver>? observers,
     bool debugLogDiagnostics = false,
     String? restorationScopeId,
+    BackButtonDispatcher? backButtonDispatcher,
```

Comment:
Is there a use case for this? I don't think we need to expose this api.

Reply:
No I'm happy with defaulting to the RootBackButtonDispatcher.